app/models/template.rb

```diff
@@ -170,6 +170,7 @@ def set_creation_defaults
       self.visibility = 1
       self.is_default = false
       self.version = 0 if self.version.nil?
+      self.links = { funder:[], sample_plan:[] } if self.links.nil?
```

Comment:
the default is set at db level on the migration file, if you don't change the type to TEXT, this line is not needed